Makefile

```
@@ -195,6 +195,43 @@ e2e-test:
 	LINODE_TOKEN=$(LINODE_TOKEN) \
 	chainsaw test e2e/test --parallel 2
 
+.PHONY: e2e-test-bgp
+e2e-test-bgp:
+	KUBECONFIG=$(KUBECONFIG_PATH) kubectl -n kube-system patch daemonset ccm-linode --patch '\
+	spec:\
+	  template:\
+	    spec:\
+	      containers:\
+	      - name: ccm-linode\
+	        args:\
+	        - --leader-elect-resource-lock=leases\
+	        - --v=3\
+	        - --secure-port=10253\
+	        - --webhook-secure-port=0\
+	        - --enable-route-controller=true\
+	        - --vpc-name=capl-cluster\
+	        - --configure-cloud-routes=true\
+	        - --cluster-cidr=10.0.0.0/8\
+			- --load-balancer-type=cilium-bgp\
+        	- --bgp-node-selector=cilium-bgp-peering=true\
+        	- --ip-holder-suffix=e2e-test
```

```suggestion
        	- --ip-holder-suffix=$(CLUSTER_NAME)
```
I would change this to use cluster name as there could be multiple PRs running e2e tests at the same time.

---

Good point! I'll make the changes